system-addon/lib/ActivityStream.jsm

```diff
@@ -16,9 +16,29 @@ const {PrefsFeed} = Cu.import("resource://activity-stream/lib/PrefsFeed.jsm", {}
 const {Store} = Cu.import("resource://activity-stream/lib/Store.jsm", {});
 const {TelemetryFeed} = Cu.import("resource://activity-stream/lib/TelemetryFeed.jsm", {});
 const {TopSitesFeed} = Cu.import("resource://activity-stream/lib/TopSitesFeed.jsm", {});
+const {DummySectionFeed} = Cu.import("resource://activity-stream/lib/DummySectionFeed.jsm", {});
 
 const REASON_ADDON_UNINSTALL = 6;
 
+// Sections, keyed by section id
+const SECTIONS = new Map([
+  ["dummy_section", {
+    feed: DummySectionFeed,
+    showByDefault: false
+  }]
+]);
+
+const SECTION_FEEDS_CONFIG = Array.from(SECTIONS.entries()).map(entry => {
+  const id = entry[0];
+  const {feed: Feed, showByDefault: value} = entry[1];
+  return {
+    name: `section.${id}`,
+    factory: () => new Feed(),
+    title: `${id} section feed`,
+    value
+  };
+});
+
```

Comment:
I think the configuration situation in this file is getting too hard to read / needs some refactoring, but I'm OK with filing a separate issue for it.